demystifying_python/demystifying_python.md

```diff
@@ -21,11 +28,30 @@ After completion of this module, learners will be able to:
 
 @end
 
-link:  https://chop-dbhi-arcus-education-website-assets.s3.amazonaws.com/css/styles.css
+good_first_module: 
+coding_required: 
```

Comment:
```suggestion
coding_required: false
```

Right? I didn't actually verify this in the module

Reply:
Oh I totally left these blank with the intention of skimming through the module to evaluate and then forgot! Doh!